src/opts.ts

```diff
@@ -93,7 +96,7 @@ async function getCacheMapFromDockerfile(dockerfilePath: string): Promise<CacheM
         // different than /var/dance-cache of course
         const target = "/var/cache-target";
 
-        cacheMap[id] = {
+        cacheMap[`${bindRoot}/${id}`] = {
```

Comment:
A breaking change because of this. I could update this to allow `bindRoot` to be `null` to restore the original behavior. However, the only scenario where the original behavior could work reliably is when the `id`s were all explicitly set which would make their source be the working directory. We could avoid breakage by making the default `cache-dir` be `.` if we want a mostly breaking change.

Reply:
I really like the approach of maintaining backward compatibility by defaulting `cache-dir` to `.`.

This way, we avoid having to release a major version with breaking changes.

Reply:
Changes are now fully backwards compatible.